.gitignore

```diff
@@ -1,6 +1,7 @@
 .vs
 *.user
 *.cache
+*.received.txt
```

Comment:
This is in the git-ignore, yet there are a lot of .received.txt files in this comment?

Reply:
If you look, I only added .verified.txt. The test works as follows:
* We specify the document we expect (verified.txt)
* The unit test generates *.received.txt
* They must be equal

Verify.Xunit is a xunit extension specific for verifying documents easier (instead of using gigantic inline strings).

Reply:
The integration tests are basically a way to test if the e-mail templates are generated you'd expect them to. Might be a bit overkill, but we can't really test IEmailSender directly (because you'd need to e-mail someone, and it'd cost us money to run the tests).